### ISSUE TITLE
Travis CI builds broken since 2 years

### ISSUE THREAD

**Issue by probonopd** [scribusproject/scribus#107]
Travis CI builds seem to be broken since 2 years, and no one seems to care?
https://travis-ci.org/scribusproject/scribus/builds#

The last successful build was  for commit 0857680:
https://travis-ci.org/scribusproject/scribus/builds/233919559

Pull request updating the AppImage build by @LizardKing has gone ignored:
https://github.com/scribusproject/scribus/pull/98

Is this repository abandoned?

**Comment by aoloe** [comment]
no, @probonopd it's not abandoned... but it will take some work from my side to get it work again and until not so long ago i did know very little about travis and appimage.

i'm creating a small program where i also use travis and appimage and now understand a bit more about it.
i wanted to to fix the scribus script build after i'm finished with the other project... but i can also try to do it before : - )
it won't be for this year, but i can try to do it soon... promised...

**Comment by probonopd** [comment]
Let me know if I can help you with Travis CI, `linuxdeployqt` and AppImage.

**Comment by probonopd** [comment]
Maybe switching to GitLab could have advantages. I am giving it a try at https://gitlab.com/probono/scribus/-/jobs/147464803 right now.

**Comment by probonopd** [comment]
Here is a test AppImage for download (available for 14 days), feedback highly welcome:
https://gitlab.com/probono/scribus/-/jobs/147883845/artifacts/raw/Scribus-277c6e5-x86_64.AppImage
